addons/customize-avatar-border/addon.json

```
@@ -0,0 +1,64 @@
+{
+  "name": "Customizable profile picture border",
+  "description": "Allows you to hide or change the outline around profile pictures in project and studio comments.",
+  "credits": [
+    {
+      "name": "CST1229",
+      "link": "https://github.com/CST1229"
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "hide_outline.css",
+      "matches": ["https://scratch.mit.edu/projects/*", "https://scratch.mit.edu/studios/*"],
+      "settingMatch": {
+        "id": "hide-outline",
+        "value": true
+      }
+    },
+    {
+      "url": "outline_color.css",
+      "matches": ["https://scratch.mit.edu/projects/*", "https://scratch.mit.edu/studios/*"],
+      "settingMatch": {
+        "id": "hide-outline",
+        "value": false
+      }
+    },
+    {
+      "url": "fill_transparent.css",
+      "matches": ["https://scratch.mit.edu/projects/*", "https://scratch.mit.edu/studios/*"],
+      "settingMatch": {
+        "id": "fill-transparent",
+        "value": true
+      }
+    }
+  ],
+  "dynamicEnable": true,
+  "dynamicDisable": true,
+  "updateUserstylesOnSettingsChange": true,
+  "injectAsStyleElt": true,
+  "settings": [
+    {
+      "name": "Hide outline",
+      "id": "hide-outline",
+      "type": "boolean",
+      "default": false
+    },
+    {
+      "name": "Fill transparency",
```

This name isn't very descriptive.
```suggestion
      "name": "Fill transparent profile pictures",
```